docs/pages/eas-update/rollouts.mdx

```diff
@@ -2,24 +2,29 @@
 title: Rollouts
 hideTOC: true
 ---
+
 import { Terminal } from '~/ui/components/Snippet';
 
+> This feature is in Developer Preview. It's not advised to use this feature in any production environments.
+
+Rollouts incrementally deploy updates on a new branch to a specific channel. With rollouts, we can roll out updates from one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to our production environment.
 
-Rollouts allow deploying a new branch to a specific channel incrementally. With rollouts, we can roll out one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to your production environment.
+To start a rollout, we use `npx eas-cli@latest` or ensure we have version \<TODO\> or above for `eas-cli`. Then we run the following command:
 
-To initiate a rollout, use the following command:
+<Terminal cmd={['npx eas-cli@latest channel:rollout [channel]']} />
 
-<Terminal cmd={["eas channel:rollout [channel]"]} />
+An interactive guide will assist us in selecting a channel, choosing a branch for the rollout, and setting the percentage of users for the rollout. To increase or decrease the rollout amount, we run the command again and choose the `Edit` option to adjust the rollout percentage.
 
-The interactive prompts will guide you through choosing a channel, a branch to begin rolling out, and the percentage of your users to rollout to. To increase or decrease the rollout amount, run the `eas channel:rollout [channel]` command again to adjust the rollout percentage.
+### Ending a rollout
 
-Once you feel confident with your new branch, or if you want to revert to your original branch, you can end the rollout with the `--end` flag:
+Two methods are available to end a rollout when we choose the `End` option in the interactive guide:
 
-<Terminal cmd={["eas channel:rollout [channel] --end"]} />
+- Republish and revert: This option can be used once we are confident with the update served from our new branch. By choosing this, we'll republish the update onto the old branch and revert users back to it.
```

Comment:
```suggestion
- **Republish and revert:** This option can be used once you are confident with the update served from the new branch. This will publish the update again on the old branch and users will be reverted back to it.
```